### ISSUE TITLE
OpenAI Assistant API

### ISSUE THREAD

**Issue by MAS-CREATIVE-LABS** [activepieces/activepieces#3163]
I think the title is self-explanatory. 
Adding the ability to converse with an assistant from Open AI using the API

**Comment by abuaboud** [comment]
I agree 100% with that, as we should deprecate beta chatbots and replaced it with excellent OpenAI piece since they are now supported Assistants. 

What is the use case? are you looking to create assistants or just using them?

**Comment by abuaboud** [comment]
For any contributors who would love to contribute please let us know, the guide to create pieces is there

https://www.activepieces.com/docs/developers/overview

**Comment by MoShizzle** [comment]
@abuaboud I can take this issue forward, I'm going to add a couple of things to the OpenAI piece including the assistants

**Comment by abuaboud** [comment]
https://github.com/activepieces/activepieces/issues/3187
